coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
LGTM. Only, it's better probably to check for inclusion, i. e. check that you have all you need. then it will not be big deal if user will have some custom files in there.